setup.py

```diff
@@ -1,40 +1,26 @@
 #!/usr/bin/env python
 # -*- encoding: utf-8 -*-
 import os.path
+
 from setuptools import setup, find_packages
 
 script_path = os.path.dirname(__file__)
 
 setup(
     name='cyclonedx-bom',
     version=open(os.path.join(script_path, 'VERSION')).read(),
-    description='CycloneDX Software Bill of Materials (SBOM) generation utility',
-    long_description=open(os.path.join(script_path, 'README.rst')).read(),
-    packages=find_packages(),
-    package_data={'cyclonedx.schema': ['bom-1.0.xsd', 'spdx.xsd']},
-    include_package_data=True,
-    zip_safe=False,
-    install_requires=[
-        'requirements_parser==0.2.0',
-        'packageurl_python==0.9.4',
-        'xmlschema==1.2.5',
-        'requests==2.25.0',
-        'packaging==20.9',
-        'jsonschema==3.2.0',
-    ],
-    entry_points={
-        'console_scripts': [
-            'cyclonedx-py = cyclonedx.client:main'
-        ]
-    },
+    url='https://github.com/CycloneDX/cyclonedx-python',
     author='Steve Springett',
     author_email='steve.springett@owasp.org',
     maintainer='Steve Springett',
     maintainer_email='steve.springett@owasp.org',
-    url='https://github.com/CycloneDX/cyclonedx-python',
+    description='CycloneDX Software Bill of Materials (SBOM) generation utility',
+    long_description=open(os.path.join(script_path, 'README.md')).read(),
+    long_description_content_type="text/markdown",
     keywords=["BOM", "SBOM", "SCA", "OWASP"],
     license='Apache-2.0',
     classifiers=[
+        'Development Status :: 4 - Beta',
```

Comment:
that is odd.
 here it is marked as beta... 

version is set to `1.0.0` in `VERSION ` file 
in the `pyproject.toml` its `1.0.0`, too 

it this was a beta, then the version should be `1.0b.0` according to 
PEP 440 - https://www.python.org/dev/peps/pep-0440/#pre-releases

Reply:
Updated - thanks @jkowalleck